### DISCUSSION TITLE
Do the new Agents allow Browser/Scraper?

### DISCUSSION THREAD

**Discussion by michnovka** [danny-avila/LibreChat#4925] [Q&A]
Reading the [official docs](https://www.librechat.ai/docs/configuration/tools), amongst plugins we had `Browser/Scraper`

I do not see this when I want to add a tool to my agent, unlike all the other plugins which do appear.

**Comment by michnovka** [comment]
If I paste the agent a link, it tells me it cannot browse or access websites, then it often goes on and hallucinates something

**Comment by danny-avila** [comment]
You need to use a tool/action that has that capability, it will not natively do so and there is no current integration for this.

**Reply by michnovka** [reply]
Is this what https://github.com/danny-avila/LibreChat/issues/4876 will bring later when released?

Also, when you mean to add a tool - thats what I meant, in older versions we had that, it is even referenced in the docs link I provided - https://www.librechat.ai/docs/configuration/tools#browserscraper

If it was removed, should it not be removed from the docs also?
